### ISSUE TITLE
Not an Issue - Idea - It's possible to use Cloud Services ( Azure ) instead local STT

### ISSUE THREAD

**Issue by Maverick87Shaka** [ciribob/DCS-SimpleTextToSpeech#2]
First of all,
Thanks for this great idea.
I'm planning to use this tool in my new server that we're going to open. We've already in place the good script for ATIS ( DATIS ) that is using Cloud service, with a really better result over the standard Windows integrated TTS.

You can consider to add cloud provider as well?

Thanks

**Comment by ciribob** [comment]
Already supports Google TTS 

run DCS-SR-ExternalAudio.exe directly with --help and you'll see some details around it

**Comment by brunonakai** [comment]
I Agree with Maverick87Shaka, microsoft's neural system is much smarter and much more natural, consider taking a look at this link:

https://azure.microsoft.com/services/cognitive-services/text-to-speech/

**Comment by ciribob** [comment]
Than Googles? Google is now supported so I dont see the need of adding a third option really

**Comment by Maverick87Shaka** [comment]
> 
> 
> Than Googles? Google is now supported so I dont see the need of adding a third option really

Yes, we've tested with DATIS different voices, comparing Google VS Azure also, result was really better on MS Azure rather than google.

Agree with the squadron we've selected en-US-Aria and en-US-Zira. Not yet tested the new one added after today newsletters.

**Comment by ciribob** [comment]
Fair enough, I'll have a look at the Microsoft api too

**Comment by Maverick87Shaka** [comment]
> 
> 
> Fair enough, I'll have a look at the Microsoft api too

When some draft will be ready I'll be happy to test it as I've done for DATIS, or also if you need a testing key to perform on your own, let me know
